boards/arm/nucleo_g071rb/doc/index.rst

```diff
@@ -133,7 +133,16 @@ Flashing
 ========
 
 Nucleo G071RB board includes an ST-LINK/V2-1 embedded debug tool interface.
-This interface is not yet supported by the openocd version included in the Zephyr SDK.
+This interface is not yet officially supported by the openocd version included in the Zephyr SDK.
+You can still update, compile and install a 'local' openocd from the official repo http://openocd.zylin.com
+and run the openocd command:
+
+.. code-block:: console
+
+   $ west flash --openocd /usr/local/bin/openocd
+
+Note that '/usr/local/bin/openocd' is your path for the freshly installed openocd, given by "$ which openocd"
+
 Instead, support can be enabled on pyocd by adding "pack" support with
```

Comment:
Can you rework a bit to have a clean transitin with the next paragraph about current pyocd support?

Reply:
changing text to explain the use of openocd interface